src/it/scala/temple/containers/PostgresSpec.scala

```diff
@@ -33,19 +33,19 @@ trait PostgresSpec extends FlatSpec with DockerTestKit with DockerPostgresServic
   // Execute a query that does not return a result
   def executeWithoutResults(query: String): Unit = execute { _.createStatement().execute(query) }
 
-  def executeWithoutResultsPrepared(preparedStatement: String, values: List[PreparedVariable]): Unit = execute { con =>
-    val prep = con.prepareStatement(preparedStatement)
+  def executeWithoutResultsPrepared(preparedStatement: String, values: List[PreparedVariable]): Unit = execute { conn =>
+    val prep = conn.prepareStatement(preparedStatement)
     values.view.zipWithIndex foreach {
       case (v, i) =>
-        val question_number = i + 1
+        val variable_number = i + 1
```

Comment:
Any particular reason for snake case? And `values.view zip (Stream from 1)` might work better, as then you’ll reduce further the chance of OBOEs (obviously removing this line)

Reply:
Then you could reduce it down to
```scala
import PreparedVariable._
val prep = conn.prepareStatement(preparedStatement)
values.view zip (Stream from 1) foreach {
  case (IntVariable(value),i) => prep.setInt(i, value)
  case (BoolVariable(value),i) => prep.setBool(i, value)
  // ...
}
```